infrastructure/main.tf

```
@@ -110,3 +110,54 @@ resource "azurerm_key_vault_secret" "draft-store-key" {
   value        = random_string.draft_encryption_key.result
   key_vault_id = data.azurerm_key_vault.ccd_shared_key_vault.id
 }
+
+////////////////////////////////
+// DB version 11              //
+////////////////////////////////
+
+module "data-store-db-v11" {
+  source          = "git@github.com:hmcts/cnp-module-postgres?ref=master"
+  product         = var.product
+  component       = var.component
+  name            = "${local.app_full_name}-postgres-db-v11"
+  location        = "${var.location}"
+  env             = "${var.env}"
+  subscription    = "${var.subscription}"
+  postgresql_user = "${var.postgresql_user}"
+  database_name   = "${var.database_name}"
+  postgresql_version = "11"
+  sku_name        = "${var.database_sku_name}"
+  sku_tier        = "GeneralPurpose"
+  sku_capacity    = "${var.database_sku_capacity}"
+  storage_mb      = "${var.database_storage_mb}"
+  common_tags     = "${var.common_tags}"
+}
+
+resource "azurerm_key_vault_secret" "POSTGRES-USER-V11" {
+  name         = "${var.component}-POSTGRES-USER-V11"
+  value        = module.data-store-db-v11.user_name
+  key_vault_id = data.azurerm_key_vault.ccd_shared_key_vault.id
+}
+
+resource "azurerm_key_vault_secret" "POSTGRES-PASS-V11" {
+  name         = "${var.component}-POSTGRES-PASS-V11"
+  value        = module.data-store-db-v11.postgresql_password
+  key_vault_id = data.azurerm_key_vault.ccd_shared_key_vault.id
+}
+
+resource "azurerm_key_vault_secret" "POSTGRES_HOST-V11" {
+  name         = "${var.component}-POSTGRES-HOST-V11"
+  value        = module.data-store-db-v11.host_name
+  key_vault_id = data.azurerm_key_vault.ccd_shared_key_vault.id
+}
+
+resource "azurerm_key_vault_secret" "POSTGRES_PORT-V11" {
+  name         = "${var.component}-POSTGRES-PORT-V11"
+  value        = module.data-store-db-v11.postgresql_listen_port
+  key_vault_id = data.azurerm_key_vault.ccd_shared_key_vault.id
+}
+
+resource "azurerm_key_vault_secret" "POSTGRES_DATABASE-V11" {
+  name         = "${var.component}-POSTGRES-DATABASE-V11"
+  value        = module.data-store-db-v11.postgresql_database
+  key_vault_id = data.azurerm_key_vault.ccd_shared_key_vault.id
```

missing a closing brace
```suggestion
  key_vault_id = data.azurerm_key_vault.ccd_shared_key_vault.id
}
```